### PR TITLE
Fix bool.from_bytes

### PR DESCRIPTION
**Pull request by hyperbora**

### CODE REVIEW

File: vm/src/obj/objint.rs [hunk 1]
```diff
@@ -233,6 +233,12 @@ impl PyInt {
     {
         if cls.is(&vm.ctx.types.int_type) {
             Ok(vm.ctx.new_int(value).downcast().unwrap())
+        } else if cls.is(&vm.ctx.types.bool_type) {
+            if PyInt::from(value).value.is_zero() {
```
Comment (coolreader18) [inline vm/src/obj/objint.rs:237]: You can use `ctx.new_bool` here, and you don't need the `PyInt::from(value). value`, that's the same thing as `value`

**Comment by hyperbora** [comment]
Thank you for your review. I revised it as you told me.

**Review by coolreader18 — APPROVED** [review]
Thanks for contributing!

File: vm/src/obj/objint.rs [hunk 2]
```diff
@@ -233,6 +233,12 @@ impl PyInt {
     {
         if cls.is(&vm.ctx.types.int_type) {
             Ok(vm.ctx.new_int(value).downcast().unwrap())
+        } else if cls.is(&vm.ctx.types.bool_type) {
+            Ok(vm
+                .ctx
+                .new_bool(!value.into().eq(&BigInt::zero()))
```
Comment (youknowone) [inline vm/src/obj/objint.rs:239]: there is also `is_zero` method

https://docs.rs/num/0.3.0/num/trait.Zero.html#tymethod.is_zero

**Pull request merged**